tests/test_utils.py

```diff
@@ -2,6 +2,7 @@
 Tests for various datasette helper functions.
 """
 
+from unittest import mock
```

Comment:
I don't think the code uses this, I'll remove it.

Reply:
Thx